web_menu_collapsible/static/src/css/menu_collapsible.css

```diff
@@ -0,0 +1,42 @@
+.oe_secondary_menu_section {
+    height: 26px;
+    cursor: pointer;
+    -webkit-user-select: none;
+    -moz-user-select: none;
+    -ms-user-select: none;
+    user-select: none;
+}
+
+.oe_secondary_menu_section.oe_menu_toggler {
+    margin-left: 18px;
+}
+
+.openerp .nav-pills li > a {
```

Comment:
this selector is very generic, it may affect other elements that are not in the menu, can't this be a more specific selector to the left menu namespace only?